manifests/registry/tekton-e2e/Readme.md

```diff
@@ -0,0 +1,6 @@
+
+Before applying those resources you nedd to create a secret with the credentials:
+
+```bash
+kubectl create secret generic e2e-registry-secret --from-literal=username=<username> --from-literal=password=<password>
```

Comment:
We can use Vault for storing the credentials and use the CR for automatic generation of secret. I've raised a PR against your fork-branch.

Reply:
thanks for contributing it! 👍